cpp/src/IceGrid/Topics.h

```diff
@@ -14,20 +13,20 @@
 namespace IceGrid
 {
 
-class ObserverTopic : public IceUtil::Monitor<IceUtil::Mutex>, public virtual Ice::Object
+class ObserverTopic : public virtual Ice::Object
```

Comment:
If this class needs to derive from Ice::Object, it would be nice to add a comment describing why.

Reply:
Hmm... I don't think it needs to.